crates/matrix-sdk/src/encryption/verification/requests.rs

```diff
@@ -38,6 +38,11 @@ impl VerificationRequest {
         self.inner.is_cancelled()
     }
 
+    /// Get the transaction id of this verification request
```

Comment:
We shouldn't mention a transaction id here if we have gone out of the way to not call this identifier a transaction id.

Reply:
I used `transaction_id` because the `ToDevice` events use it. And for me it was difficult to figure out what the flow id was.